code/modules/jobs/job_types/maintenance_technician.dm

```diff
@@ -0,0 +1,56 @@
+/datum/job/maintenance_technician
+	title = JOB_MAINTENANCE_TECHNICIAN
+	description = "Fix everything you are told to fix, \
+		hope nobody notices when you accidentally break something."
+	department_head = list(JOB_CHIEF_ENGINEER)
+	faction = FACTION_STATION
+	total_positions = 3
+	spawn_positions = 3
+	supervisors = "the engineering department and Chief Engineer"
+	exp_granted_type = EXP_TYPE_CREW
+	config_tag = "MAINTENANCE_TECHNICIAN"
+
+	outfit = /datum/outfit/job/maintenance_technician
+	plasmaman_outfit = /datum/outfit/plasmaman/engineering
+
+	paycheck = PAYCHECK_CREW
+	paycheck_department = ACCOUNT_ENG
+
+	liver_traits = list(TRAIT_ENGINEER_METABOLISM)
+
+	display_order = JOB_DISPLAY_ORDER_STATION_ENGINEER
+	bounty_types = CIV_JOB_ENG
+	departments_list = list(
+		/datum/job_department/engineering,
+		)
+
+	family_heirlooms = list(/obj/item/clothing/head/utility/hardhat, /obj/item/screwdriver, /obj/item/wrench, /obj/item/weldingtool, /obj/item/crowbar, /obj/item/wirecutters)
+
+	mail_goodies = list(
+		/obj/item/storage/box/lights/mixed = 20,
+		/obj/item/lightreplacer = 10,
+		/obj/item/holosign_creator/engineering = 8,
+		/obj/item/wrench/bolter = 8,
+		/obj/item/clothing/head/utility/hardhat/red/upgraded = 1
```

Comment:
```suggestion
		/obj/item/clothing/head/utility/hardhat/red/upgraded = 1,
```

Reply:
I literally copy pasted engineer code, it is the same there lmao

Reply:
its bad there too